src/components/post/comments/Reply.vue

```diff
@@ -81,7 +103,7 @@ onMounted(() => {
 					style="top: 40px; right: 0px"
 				>
 					<!-- Delete -->
-					<button class="focus:outline-none text-negative flex">
+					<button class="focus:outline-none text-negative flex" @click="removeReply()">
```

Comment:
@dxphilo can you hide the comment once it's deleted? Check out how I do it on the comment component. Basically, it's an `isDeleted` ref that I change to true after teh delete goes through. Then set that on the reply v-if="!isDeleted"